Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -62,6 +64,7 @@ type Props = $ReadOnly<{|
   rejectResponderTermination?: ?boolean,
   testID?: ?string,
   touchSoundDisabled?: ?boolean,
+  style?: ?ViewStyleProp,
```

Comment:
Is this style prop getting passed through correctly in TouchableWithoutFeedback? I would think it needs to get passed to some underlying View to work.

Reply:
yes, will remove this from here and directly add styles within the component file where `TouchableWithoutFeedback` is used.

Reply:
You mean to say like this:
```
               // passing styles to the underlying View //
              <View accessible={true} style={styles.touchableContainer}> 
                <Text style={{fontSize: 25}}>Hello</Text>
              </View>
            </TouchableWithoutFeedback>
```